crates/primitives/src/env.rs

```diff
@@ -174,16 +174,6 @@ impl Env {
                     return Err(InvalidTransaction::BlobVersionNotSupported);
                 }
             }
-
-            // ensure the total blob gas spent is at most equal to the limit
-            // assert blob_gas_used <= MAX_BLOB_GAS_PER_BLOCK
-            let num_blobs = self.tx.blob_hashes.len();
-            if num_blobs > MAX_BLOB_NUMBER_PER_BLOCK as usize {
```

Comment:
should we still keep this check if we're between cancun and prague?

Reply:
It was unnecessary even in Prague. This check must be done on the whole block, not only one tx.